jac-uk/digital-platform
# Verify recaptcha token

## What's included?
Implement Google reCAPTCHA verification. Need to set config in different environment:
- `firebase functions:config:set google_recaptcha.url=https://www.google.com/recaptcha/api/siteverify`
- `firebase functions:config:set google_recaptcha.secret=<SECRET>`

Reference: https://developers.google.com/recaptcha/docs/verify

Note: this PR is related to [#901 Anti-automation protection](https://github.com/jac-uk/apply/pull/905)

## Who should test?
✅ Developers

## How to test?
1. Run cloud functions.
2. Run apply site (branch: [feature/901-anti-automation-protection](https://github.com/jac-uk/apply/tree/feature/901-anti-automation-protection)).
3. Check functionalities on `Sign In` and `Forgotten password` pages.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context

## Related permissions
- No permission changes required

---
PREVIEW:DEVELOP
